lib/broccoli/angular-broccoli-less.js

```diff
@@ -0,0 +1,57 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+let less;
+try {
+  less = require(`${process.env.PWD}/node_modules/less`);
```

Comment:
Does this still work if you're inside a directory (like in a component)? It seems really fragile and makes assumptions.